src/compressed_tensors/quantization/quant_config.py

```diff
@@ -165,7 +165,7 @@ def to_dict(self):
 
     @staticmethod
     def from_pretrained(
-        model: Module, format: Optional[str] = None
+        model: Module, format: Optional[Union[str, list]] = None
```

Comment:
Ideally format would just be a list and not nullable, but no rush to worry about that now